### ISSUE TITLE
Choice of hyperparameter (alpha, kernel, Acquisition function, xi, kappa)

### ISSUE THREAD

**Issue by mkersche17** [fmfn/BayesianOptimization#79]
First of all, thank you very much fernando for creating this nice implementation of bayesian optimization. I recently became aware of bayesian optimization and I think it is a good alternative to methods like grid search or random search. 
However, I struggle to find good bayesian optimization parameters (i.e. alpha) for my given problem. I train a neural network, which takes several hours for one iteration, and want to optimize parameters like hidden layer count, neurons per layer, learning rate, dropout value, lambda of l2 regularization, etc. 
Because I'm new in bayesian optimization, I have completely no clue, how to set the bo parameters.  When I expect a very thin maximum (in terms of: good results with 2 hidden-layer bad results using 3), how should I decrease alpha? set it to 1e-4 or 1e-3 or 1e-2?
In gerneral when you work with machine learning, you often get advices like, use the ReLU activiation, this often works good or use a learning rate of 1e-3 and so on. However, for bayesian optimization I have not found any recommendations. 
Would it be possible to create a thread, where users can share their experience? So people can describe their task (i.e. time series classification, N training examples, M inputs, O outputs, count of parameters which should be optimized) and which alpha, kernel, aquisition function, xi, kappa worked the best. 
This can help to make the project more applicable.

**Comment by fmfn** [comment]
Default parameters should do a pretty good job out of the box. Sometimes increasing `alpha` is needed, specially in cases with discrete parameters (such as number of layers). The default kernel is the one suggested by the authors of the original paper this package is based on, same goes for acquisition function. Xi and Kappa control exploration vs exploitation, and are up to you to tune as you play around with the optimization a couple times.

**Comment by Amir-Arsalan** [comment]
@fmfn Is there a upper bound for `xi` when using Expected Improvement?

**Comment by Amir-Arsalan** [comment]
@fmfn In my problem, the variable bounds is `[0, 1)` but I'm setting `xi` to a pretty high value (`4`). I just looked at [this](https://github.com/Amir-Arsalan/BayesOpt/blob/master/bayes_opt/helpers.py#L110-L113)  which made me think I might be doing something wrong. I'm not entirely following what the `_ei`  function is doing but it looks a bit counter-intuitive to have such a high value for `xi` for my problem given that `xi` is getting subtracted from `mean` and `y_max`. Could you please clarify on this?
